src/main/java/com/snack/news/domain/news/News.java

```diff
@@ -4,10 +4,8 @@
 import com.snack.news.domain.category.Category;
 import com.snack.news.domain.tag.Tag;
 import com.snack.news.domain.topic.Topic;
-import lombok.Builder;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.ToString;
+import com.snack.news.dto.NewsDto;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'com.snack.news.dto.NewsDto'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4554642)